### ISSUE TITLE
FNX3-15839 ⁃ [Bug] Fenix crashes when trying to load ghacks.net

### ISSUE THREAD

**Issue by cadeyrn** [mozilla-mobile/fenix#1820]
### Steps to reproduce
1. Try to load the website ghacks.net

### Expected behavior
Fenix opens the website.

### Actual behavior
Fenix crashes. It's the only website where I can reproduce the crash.

```
2019-04-19 18:24:59.046 14884-14900/org.mozilla.fenix.debug E/WM-WorkerWrapper: Work [ id=8d05415e-b4b1-48ce-9b52-b33e999d2696, tags={ Immediate, mozilla.components.feature.sync.WorkManagerSyncWorker, Common } ] failed because it threw an exception/error
    java.util.concurrent.ExecutionException: kotlin.KotlinNullPointerException
        at androidx.work.impl.utils.futures.AbstractFuture.getDoneValue(AbstractFuture.java:516)
        at androidx.work.impl.utils.futures.AbstractFuture.get(AbstractFuture.java:475)
        at androidx.work.impl.WorkerWrapper$2.run(WorkerWrapper.java:289)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
        at java.lang.Thread.run(Thread.java:764)
     Caused by: kotlin.KotlinNullPointerException
        at mozilla.components.feature.sync.WorkManagerSyncWorker.doWork(WorkManagerSyncDispatcher.kt:216)
        at androidx.work.CoroutineWorker$startWork$1.invokeSuspend(CoroutineWorker.kt:66)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:32)
        at kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:234)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:594)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.access$runSafely(CoroutineScheduler.kt:60)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:742)
```

### Device information
* Android device: HTC U11 / Android 8.0
* Fenix version: Fenix revision af613962e550c1b9ab9931b12ee1c30f88c88ff6

**Comment by vesta0** [comment]
@cadeyrn I am unable to reproduce this. Are you still seeing the crash?

**Comment by cadeyrn** [comment]
Yes, it still crashes with latest build from master.

**Comment by vesta0** [comment]
@cadeyrn you are right. I didn't get a crash yesterday but I am getting one today.

**Comment by cadeyrn** [comment]
Current Gecko version is 68.0.20190422094240. Update to 68.0.20190423095327 or newer will fix the crash.

**Comment by Lululalu** [comment]
It doesn't reproduce for me anymore.

**Comment by cadeyrn** [comment]
@Lululalu Then test again. Maybe you were lucky in your previous test, but I am pretty sure it will crash again the next time. The crash is not fixed and it can't be fixed. As I already explained it's a bug in the used GeckoView version and there was no GV update. I compiled a new build from master and - as expected - it still crashes.

\cc @pocmo / @ncalexan / @colintheshots since you did GV updates in the past: Could you or someone else please update the GV version of Fenix?

See my previous comment: The used GV version has a bug which is already fixed since 8 days and this bug causes crashes on a few websites. So it should definitively be fixed before the Beta launch.

The bad patch was introduced in https://bugzilla.mozilla.org/show_bug.cgi?id=1536511 and this is the Bugzilla ticket about the crash: https://bugzilla.mozilla.org/show_bug.cgi?id=1545023. As you can see there and in the duplicates not only ghacks.net should be affect by this bug but also other websites (ghacks.net is the only website I tested).

**Comment by cadeyrn** [comment]
Fixed by GV update in #2264.
